pkg/controller/controller_test.go

```diff
@@ -2295,8 +2297,10 @@ func TestCatalogConversionMultipleServiceClasses(t *testing.T) {
 		// For service1 make sure we have service level metadata with field1 = value1 as the blob
 		// and for service1 plan s1plan2 we have planmeta = planvalue as the blob.
 		if sc.Name == "service1" {
-			if *sc.Description != "service 1 description" {
-				t.Fatalf("Expected service1's description to be \"service 1 description\", but was: %s", sc.Description)
+			if sc.Description == nil {
+				t.Fatalf("Expected service1's description to be \"service 1 description\", but was nil")
+			} else if *sc.Description != "service 1 description" {
+				t.Fatalf("Expected service1's description to be \"service 1 description\", but was: %s", *sc.Description)
```

Comment:
I _think_ this works:
if plan.Description == nil || *plan.Description != planDescription {
    t.Fatalf("Expected... 

But then the print will be wrong, so I am not aware (which of course doesn't mean there is not :) )